### PR TITLE
NEWS: update 2.18.0

### PR DESCRIPTION
**Pull request by yasminvalim**
Updating the Ignition documentation for the v2.18.0 release.

This step is part of the release checklist. You can take a look at the [current checklist](https://github.com/coreos/ignition) for this issue.

### CODE REVIEW

**Review by jmarrero — CHANGES_REQUESTED** [review]
I see on a previous release you updated this line https://github.com/coreos/ignition/pull/1822/files#diff-335380bcba1c52cf710d58b3799901d23af56212720c334ab4ef42786efc9a4fR7 too, do we need that too?:
https://github.com/coreos/ignition/pull/1756/files

**Comment by yasminvalim** [comment]
> I see on a previous release you updated this line https://github.com/coreos/ignition/pull/1822/files#diff-335380bcba1c52cf710d58b3799901d23af56212720c334ab4ef42786efc9a4fR7 too, do we need that too?: https://github.com/coreos/ignition/pull/1756/files

Oops! Nice catch. I added 2.19.0 as unreleased now. Thanks :)

File: docs/release-notes.md [hunk 1]
```diff
@@ -4,7 +4,17 @@ nav_order: 9
...
+
+### Breaking changes
+
+### Features
+
+### Changes
+
+### Bug fixes
+
+## Upcoming Ignition 2.18.0 (2024-03-01)
```
Comment (jmarrero) [inline docs/release-notes.md:17]: Should it be `## Ignition 2.18.0 (2024-03-01)` ?

**Review by jmarrero — APPROVED** [review]
lgtm

**Review by jmarrero — APPROVED** [review]

**Pull request merged**